packages/react-flow-editor/src/Editor/components/Output/index.tsx

```diff
@@ -18,15 +18,15 @@ type Props = {
 
 export const Output: React.FC<Props> = React.memo(({ nodeId, nodeState, output }) => {
   const { OutputComponent } = useEditorContext()
-  const { zoomContainer } = useRectsContext()
+  const { zoomContainerRef } = useRectsContext()
   const hoveredNodeId = useStore(HoveredNodeIdAtom)
   const dragItem = useStore(DragItemAtom)
 
   const startNewConnection = (e: React.MouseEvent<HTMLElement>) => {
-    const zoomRect = zoomContainer.getBoundingClientRect()
+    const zoomRect = getDOMRect(zoomContainerRef.current)
```

Comment:
Name without get 
zoomRect uses once, we should remove that

Reply:
not once